ee/hogai/graph/billing/prompts.py

```diff
@@ -0,0 +1,132 @@
+BILLING_CONTEXT_PROMPT = """
+<billing_context>
+# Billing & Subscription Information
+The user has {{subscription_level}} subscription{{#billing_plan}} ({{billing_plan}}){{/billing_plan}}.
+
+{{#has_active_subscription}}
+## Current Subscription Status
+- Active subscription: Yes
+{{#startup_program_label}}
+- Startup program: {{startup_program_label}}
+{{/startup_program_label}}
```

Comment:
I see you're adding to context whether they were in startup program previously - it can be useful context (e.g. to determine eligibility for startup program)